gateway/conf/nginx.conf.liquid

```diff
@@ -34,6 +34,7 @@ events {
 
 http {
   sendfile           on;
+  sendfile_max_chunk 512k;
```

Comment:
I've read the description in https://nginx.org/en/docs/http/ngx_http_core_module.html#sendfile_max_chunk but it's not clear to me in which cases this could be worse than the default. What scenario did you test and what perf numbers did you get?

Reply:
Mainly, when large bodies are send. We have an Hyperfoil testcase where a lot of large bodies are in place. With this change, we got around 5% more rps.

Reply:
:+1:
Do you know if there's a performance penalty for use cases with small or no request bodies?

Reply:
no penalty, because is write in chunks  instead of fill the chunk, so not issues at all.